float64 x
float64 y
float64 z
---
float64 theta_one
float64 theta_two
float64 d_three
